internal/provider/provisioner_key_resource_test.go

```diff
@@ -0,0 +1,96 @@
+package provider
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/coder/terraform-provider-coderd/integration"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccProvisionerKeyResource(t *testing.T) {
+	if os.Getenv("TF_ACC") == "" {
+		t.Skip("Acceptance tests are disabled.")
+	}
+	ctx := context.Background()
+	client := integration.StartCoder(ctx, t, "license_acc", true)
+	orgs, err := client.Organizations(ctx)
+	require.NoError(t, err)
+	firstOrg := orgs[0].ID
+
+	license := os.Getenv("CODER_ENTERPRISE_LICENSE")
+	if license == "" {
+		t.Skip("No license found for license resource tests, skipping")
+	}
```

Comment:
This check & skip already happens in `StartCoder` when is `useLicense` is true, so you can get rid of it. The reason we have it for the license resource tests is because we're adding the license ourselves via the new resource.